reporting/accuracy/plot_jacobi_theta1.cpp

```diff
@@ -0,0 +1,31 @@
+
+#include <iostream>
+#include <boost/math/tools/ulps_plot.hpp>
+#include <boost/core/demangle.hpp>
+#include <boost/math/special_functions/jacobi_theta.hpp>
+
+using boost::math::tools::ulps_plot;
+
+int main() {
+    using PreciseReal = long double; // boost::multiprecision::float128;
+    using CoarseReal = double;
+
+    auto jacobi_theta1_coarse = [](CoarseReal z) {
+        return boost::math::jacobi_theta1<CoarseReal>(z, 0.5);
+    };
+    auto jacobi_theta1_precise = [](PreciseReal z) {
+        return boost::math::jacobi_theta1<PreciseReal>(z, 0.5);
+    };
+
+    std::string filename = "jacobi_theta1_" + boost::core::demangle(typeid(CoarseReal).name()) + ".svg";
+    int samples = 2500;
+    int width = 1100;
+    PreciseReal clip = 100;
+    auto plot = ulps_plot<decltype(jacobi_theta1_precise), PreciseReal, CoarseReal>(jacobi_theta1_precise, -10.0, 10.0, samples);
+    plot.clip(clip).width(width);
+    std::string title = "jacobi_theta1 ULP plot at " + boost::core::demangle(typeid(CoarseReal).name()) + " precision";
+    //plot.title(title);
```

Comment:
Is the title string unused?

Reply:
I just copy-pasted this from `test_agm.cpp`, which has the same line commented out.

Reply:
Note that if you don't need a title that's fine; but you'll need to label it in the docs.

Reply:
Bah then I need to fix that in `test_agm.cpp`. . .

Reply:
Uncommented these lines, resolving